docs/docsite/rst/reference_appendices/glossary.rst

```diff
@@ -250,6 +250,11 @@ when a term comes up on the mailing list.
         Ansible uses JSON for return data from remote modules.  This allows
         modules to be written in any language, not just Python.
 
+    Keyword
+        The main expressions that make up Ansible, which apply to playbook objects
+        (Play, Block, Role and Task). For example 'vars:' is a keyword that lets
```

Comment:
``vars`` instead of 'vars:'

Reply:
i'm trying to be explicit to the form used in play, including the :